python/taichi/core/primitive_types.py

```diff
@@ -0,0 +1,60 @@
+from taichi.core.util import ti_core
```

Comment:
This is probably a special case where we need to import `ti_core` directly, because the rest of the type definitions are directly executed during the import. It doesn't make a difference if we use `util.ti_core` here.